modules/DMZ/data_kit/process_folder.py

```diff
@@ -0,0 +1,42 @@
+from PIL import Image  # image operations
+import os     # operations on directories
+import imghdr # image file recognition
+import numpy  # numpy arrays
+
+# Sample Paths
+# /mnt/c/Users/Owner/Pictures/SamplePictures/
+# /mnt/c/Users/Tayo/Documents/SE2CS4365/TestImages/
+
+# list of image types
+imgfiles = ['rgb', 'gif', 'pbm', 'pgm', 'ppm', 'tiff', 'rast', 'xbm', 'jpeg', 'jpg', 'bmp', 'png', 'webp', 'exr']
+
+# Splitting up this module into smaller ones
+
+# Given a filepath and size, resizeFolder will enter the folder
+# and edit the files to be square, with lxw as sizexsize
+def resize_folder(path, size):
+    dirs = os.listdir(path)  # store the path.
+
+    for item in dirs:
```

Comment:
This code looks very similar to the code in the next function. Can you modify these two functions so that there is less duplicate code?